apps/ejabberd/include/ejabberd_odbc.hrl

```diff
@@ -0,0 +1 @@
+-define(QUERY_TIMEOUT, 5000).
```

Comment:
There's `ejabberd.hrl`, although I don't remember why it's included now. Still, we could move the definition there. Alternatively we can add `Timeout` parameter to `query` callback and only have the definition in `ejabberd_odbc` module. I think I like this option.